### DISCUSSION TITLE
Add new messages

### DISCUSSION THREAD

**Discussion by MariLehm** [ros2-rust/ros2_rust#42] [Q&A]
Hello,
I am new to the Rust and ROS2 world and need help using a new message stored in rosidl_generator_rs/msg.
The colcon build failed: It fails with the information that it cannot find my newly created message under std_msgs :: msg ::NewMessage.msg

To be honest, I haven't yet understood rosidl_generator_rs 100%, hence the question.

Can you give me a short lease on how I can add a new message to ros2_rust?

I would be very grateful for your help.
Kind regards,
Marius

**Comment by esteve** [comment]
We now have examples and tutorials on how to add new messages.
